### DISCUSSION TITLE
There should be a way to validate an API meets the standards.

### DISCUSSION THREAD

**Discussion by adrenallen** [freightapis/ssc#7] [Ideas]
Currently, there is no way to ensure an implementation of the standard is actually following the standard short of manually reviewing all endpoints case by case.

It would be beneficial to the SSC as well as supporters, to have a way to obtain certifications of their API..

This could be accomplished via something like an [official postman package](https://www.postman.com/product/what-is-postman/) or some other automated request process to determine if endpoints meet the standard.

A way to become verified against the standards will also open the opportunity for third parties to build re-usable libraries in various programming languages, which will further decrease the level of effort to integrate with someone if they follow best practices/this standard.

Without an easy way to validate an API as "following the standard", we will likely see many flavors the standard in practice. And ultimately implementation will still be case by case.

**Comment by miller79** [comment]
I appreciate your suggestion. With the first version of the specification released and as companies begin to implement it, we agree that having an official postman package for certification is for the best. We also expect this future and hope with Github that others can help by sharing their implementations and test suites that everyone can use.
